graphlearn/examples/tf/trainer.py

```diff
@@ -209,6 +275,56 @@ def train_and_evaluate(self, train_iterator, test_iterator, loss, test_acc, opti
     self.train(train_iterator, loss, optimizer, learning_rate, epochs, hooks, **kwargs)
     self.test(test_iterator, test_acc, hooks, **kwargs)
 
+  def save_node_embedding_bigdata(self, sess, save_iter, save_ids, save_emb, save_file, block_max_lines):
+      print('Start saving embeddings...')
+      # assume block_max_lines >> batch_size
+      assert block_max_lines >= 10000, 'block_max_lines must >= 10000, it is {} now'.format(block_max_lines)
```

Comment:
Can we fall back to the original `save_node_embedding` function here instead of adding an assertion to support the both large-scale and small data?

Reply:
> 

Hello, I have updated the code to use save_node_embedding when batch_size > block_max_lines. In fact, I have reconstructed the save embedding function to make it clearer. I also encountered an error when using save_node_embedding, so I made some changes to certain lines in it. I have summarized all the changes in the first paragraph(first floor), please refer to it for more information. Thank you for your advice.